### ISSUE TITLE
Dropbox menu is useless

### ISSUE THREAD

**Issue by memeplex** [ubuntu/gnome-shell-extension-appindicator#138]
This extension shows a substitute for the original dropbox menu. The replacement applies the right theme, but that's the only thing that it gets right:

1. I've to first double click the icon or the menu won't appear on single click. Once I double clicked it, a single click does open the menu.

2. Further double clicking will open two menus: the original one and its replacement. The original one is hard to get rid of, once opened.

3. The "Recently changed files" submenu of the substitute menu is always empty.

I think this extension is trying to do too much. Why not just allowing the original ---functional tough bad themed--- menu to pop up instead of a nice but useless one?

**Comment by treba123** [comment]
Probably a duplicate of #93 
Concerning the last question: the extension is not trying to do too much. It just implements a new API and not the old one because it's messy and therefore got replaced. Applications and Toolkits (QT in this case I guess) have to properly support the API, that's all.

**Comment by Locke** [comment]
It seems to me that the issues have been fixed with this recent beta update of the Dropbox client: https://www.dropboxforum.com/t5/Desktop-client-builds/Beta-Build-64-3-120/m-p/317429

![image](https://user-images.githubusercontent.com/53905/50718100-a7e85d80-108c-11e9-8086-103e2f99d3fd.png)

**Comment by mattia-b89** [comment]
Issue still here (archlinux, gnome 3.32, dropbox 70.4.93)

**Comment by schakalakka** [comment]
Are there any news about a fix? Because I can't expand the AppIndicator menu at all (the icon is there but clicking does nothing) and Dropbox is not syncing and I don't know why. :/

Using Manjaro/Gnome (3.32.2) here.

**Comment by chrisgraham** [comment]
> It seems to me that the issues have been fixed with this recent beta update of the Dropbox client: https://www.dropboxforum.com/t5/Desktop-client-builds/Beta-Build-64-3-120/m-p/317429
> 
> ![image](https://user-images.githubusercontent.com/53905/50718100-a7e85d80-108c-11e9-8086-103e2f99d3fd.png)

Unfortunately the latest beta is not fixing it for me (82.3.138).

**Comment by layolayo** [comment]
Same here - had this issue for nearly a year now. Very frustrating, clicking the 'Recently Changed Files' shows the list for a fraction of a second then disappears.

**Comment by coxackie** [comment]
yeah, 'Recently Changed Files' does not work, neither does the "Snooze notifications" submenu. Come on, this has been going on for too long... no ideas?

**Comment by layolayo** [comment]
My laptop and desktop running 19.10 are both suffering this annoying bug... Any ideas anyone? Any hope for a fix...?

**Comment by daniloquispe** [comment]
> > Are there any news about a fix? Because I can't expand the AppIndicator menu at all (the icon is there but clicking does nothing) and Dropbox is not syncing and I don't know why. :/
> > Using Manjaro/Gnome (3.32.2) here.
> 
> Solved by installing libappindicator-gtk3 from the official repo.

Hello:

Same problem here... I tried this solution and works partially:

- "Original" Dropbox menu is gone, and "GNOME Shell style" menu displays pops in/out OK
- Submenus still doesn't work

I use...
- Manjaro 20.0.1
- GNOME Shell 3.36.2 (with KStatusNotifierItem/AppIndicator Support extension enabled)
- Dropbox 98.3.151
